include/l3.h

```diff
@@ -81,11 +124,11 @@ extern "C" {
 #endif
 
 #ifdef L3_LOC_ENABLED
-void l3__log_simple(const loc_t loc, const char *msg,
-                    const uint64_t arg1, const uint64_t arg2);
+void l3__log_simple(const char *msg, const uint64_t arg1, const uint64_t arg2,
+                    const loc_t loc);
```

Comment:
Arg rearrangement was needed so that we can invoke `printf()` with just :

`const char *msg, const uint64_t arg1, const uint64_t arg`

as printf() does not know aything about `loc_t loc` argument.

Reply:
again, the `format` attribute allows for this.